### DISCUSSION TITLE
GitHub Student Developer Pack Access Issue

### DISCUSSION THREAD

**Discussion by Mohiit70** [community/community#102646] [GitHub Education]
### Hi everyone,

**Problem:**

I'm unable to access the **GitHub Student Developer Pack** because the original account associated with my college ID was deleted. When attempting to apply with my current account, I receive an error stating that the pack was already used with the deleted account. Despite contacting GitHub support through various channels, I have not received a response or solution.

**Additional Information:**

I created an issue regarding this problem before August 2023, but it remains unresolved. Despite sending emails to GitHub support, I have not received a response. The support ticket https://support.github.com/ticket/personal/0/2276241 is incorrectly linked to my primary account "Mohiit70" instead of the original account "MohitBisht700."

![Screenshot 2024-01-29 204157](https://github.com/community/community/assets/120316966/a04fe42f-a1e6-4ee7-a76b-ef2d9403322a)

![Screenshot 2024-01-29 202736](https://github.com/community/community/assets/120316966/4ac2ed4f-98c9-42fb-baeb-5ebfd6deb051)

**Comment by ravindudil5han** [comment]
I recommend trying to reach out to GitHub Support again, providing clear details about the issue, including the incorrect linking of the support ticket to your primary account and the persistence of the problem despite your previous efforts. Be sure to include your current account details and any relevant information that could assist in resolving the issue. If possible, mention the previous support ticket number for reference. Persistence and clarity in communication may increase the chances of a timely resolution.
